mozfun/mode/last/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+description: 'Returns the most frequently occuring element in an array.  In the case
```

Comment:
`mode` seems to specific to me to be a reasonable dataset-level name. I'd like us to think more about what the logical grouping would be here for functions like `mode_last`.